recipes/gvisor/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set provider = "gvisor" %}
+{% set goname = "github.com/google/"+provider.lower() %}
+{% set version = "20200323.0" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/{{ provider }}/archive/release-{{ version }}.tar.gz
+  sha256: 4a4b0d42ef741cfdb831ebc05ef6f483217515f6d186abc96af6fea3d1312353
+  patches:
+    - syscalls_rmv_boottime.patch
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+  ignore_run_exports:
+    - libstdcxx-ng
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ compiler('go') }}
+    - git
+    - bazel
+
+test:
+  commands:
+    - test //pkg/...
+    - test //runsc/...
+  source_files:
+    - pkg/
+    - runsc/
```

Comment:
The test section is a little bit problematic here.
  - What does `test //pkg/...` do? Did we mean `go test` ?
  - If we bring the entire `pkg` and `runsc` directories as source_files then the end go-package will be really big. Can we move that type of test to build.sh? 
  - Add a functional test, like `runsc --help` or equivalent, to the meta.yaml